Proiect/new-evaluation.css

```diff
@@ -67,5 +80,64 @@
 		}
 
 		li {
-			display: inline;
+			float: left;
+			text-align: left;
+			width: 16%;
+			margin: 2px;
+			min-height: 36px;
+			font-family: Arial;
+			font-size: 15px;
+		}
+
+		#submit {
```

Comment:
Just for precaution, while it's usually easiest to rely on IDs, it's often a bad choice for CSS in the long term, since styles against IDs are notoriously difficult to override, by design. In short, just make sure you don't develop a habit of over-reliance on styling against IDs, but rather choose class names (or even tag names) wherever applicable.